README.md

```diff
@@ -8,8 +8,89 @@ User documentation for the SecureDrop Workstation.
 - Run `make docs` to start a live build of the documentation at `http://127.0.0.1:8000`
 - Edit RST files under the `docs` directory - your changes will be reflected in the live build
 
+## Detailed Setup instructions for macOS
+
+There are multiple ways to set your virtual environment; here we will use
+`pyenv` and `pyenv-virtualenv` installed by `brew`.
+
+### Set up pyenv and pyenv-virtualenv
+
+From the Terminal, run
+`brew update && brew upgrade && brew install pyenv pyenv-virtualenv`.
+
+You may want to read more about [pyenv](https://github.com/pyenv/pyenv) and
+[pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv).
+
+To ensure that these tools run properly, add the following lines to your
+`~/.bash_profile` (or `~/.zprofile`) by opening that file in your favourite
+text editor (you can use `nano ~/.bash_profile` to work on the command-line):
+
+```
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+```
+
+### Fix brew errors
+
+If you run `brew doctor`, you will see the following warning:
+
+```
+Warning: "config" scripts exist outside your system or Homebrew directories.
+`./configure` scripts often look for *-config scripts to determine if
+software packages are installed, and what additional flags to use when
+compiling and linking.
+
+Having additional scripts in your path can confuse software installed via
+Homebrew if the config script overrides a system or Homebrew provided
+script of the same name. We found the following "config" scripts:
+
+```
+
+You can ignore this warning, or use one of the two possible fixes documented
```

Comment:
If the error is just Homebrew noise and can be ignored, do we need this section?

Reply:
I think it's a bit unfortunate if our instructions make `brew doctor` seem broken and we don't acknowledge it - but I have moved this to the end so that it doesn't interrupt the setup flow - see 6d1db2b